core/services/keeper/registry_synchronizer_process_logs.go

```diff
@@ -169,18 +169,18 @@ func (rs *RegistrySynchronizer) handleUpkeepPerformedLogs(done func()) {
 
 func (rs *RegistrySynchronizer) handleUpkeepPerformed(broadcast log.Broadcast) {
 	txHash := broadcast.RawLog().TxHash.Hex()
-	logger.Debugw("RegistrySynchronizer: processing UpkeepPerformed log", "jobID", rs.job.ID, "txHash", txHash)
+	rs.logger.Debugw("RegistrySynchronizer: processing UpkeepPerformed log", "txHash", txHash)
 	was, err := rs.logBroadcaster.WasAlreadyConsumed(rs.orm.DB, broadcast)
 	if err != nil {
-		logger.Warn(errors.Wrapf(err, "RegistrySynchronizer: unable to check if log was consumed, jobID: %d", rs.job.ID))
+		rs.logger.WithError(err).Warn("RegistrySynchronizer: unable to check if log was consumed")
```

Comment:
We can probably remove the "RegistrySynchronizer: "... portion of these error messages now, since the named logger already specifies this is for the registry synchronizer.

Reply:
Done